core/controlplane/config/templates/cloud-config-controller

```diff
@@ -781,108 +781,111 @@ write_files:
+        done
+        echo kube-proxy stared. apiserver should be responsive again.
+      fi
+
       mfdir=/srv/kubernetes/manifests
 
       {{ if .UseCalico }}
       /bin/bash /opt/bin/populate-tls-calico-etcd
-      kubectl apply -f "${mfdir}/calico.yaml"
+      applyall "${mfdir}/calico.yaml"
       {{ end }}
 
       {{ if .Experimental.NodeDrainer.Enabled }}
-      for manifest in {kube-node-drainer-ds,kube-node-drainer-asg-status-updater-de}; do
-          kubectl apply -f "${mfdir}/$manifest.yaml"
-      done
+      applyall "${mfdir}/{kube-node-drainer-ds,kube-node-drainer-asg-status-updater-de}".yaml"
       {{ end }}
 
       # Secrets
-      kubectl apply -f "${mfdir}/kubernetes-dashboard-se.yaml"
+      applyall "${mfdir}/kubernetes-dashboard-se.yaml"
 
       # Configmaps
-      for manifest in {kube-dns,kube-proxy}; do
-        kubectl apply -f "${mfdir}/$manifest-cm.yaml"
-      done
+      applyall "${mfdir}"/{kube-dns,kube-proxy}"-cm.yaml"
 
       # Service Accounts
-      for manifest in {kube-dns,heapster,kube-proxy,kubernetes-dashboard,metrics-server}; do
-          kubectl apply -f "${mfdir}/$manifest-sa.yaml"
-      done
+      applyall "${mfdir}"/{kube-dns,heapster,kube-proxy,kubernetes-dashboard,metrics-server}"-sa.yaml"
 
       # Install tiller by default
-      kubectl apply -f "${mfdir}/tiller.yaml"
+      applyall "${mfdir}/tiller.yaml"
 
 {{ if .KubeDns.NodeLocalResolver }}
       # DNS Masq Fix
-      kubectl apply -f "${mfdir}/dnsmasq-node-ds.yaml"
+      applyall "${mfdir}/dnsmasq-node-ds.yaml"
 {{ end }}
 
       # Deployments
-      for manifest in {kube-dns,kube-dns-autoscaler,kubernetes-dashboard,{{ if .Addons.ClusterAutoscaler.Enabled }}cluster-autoscaler,{{ end }}heapster{{ if .KubeResourcesAutosave.Enabled }},kube-resources-autosave{{ end }},metrics-server}; do
-          kubectl apply -f "${mfdir}/$manifest-de.yaml"
-      done
+      applyall "${mfdir}"/{kube-dns,kube-dns-autoscaler,kubernetes-dashboard,{{ if .Addons.ClusterAutoscaler.Enabled }}cluster-autoscaler,{{ end }}heapster{{ if .KubeResourcesAutosave.Enabled }},kube-resources-autosave{{ end }},metrics-server}"-de.yaml"
 
       # Daemonsets
-      for manifest in {kube-proxy,}; do
-          kubectl apply -f "${mfdir}/$manifest-ds.yaml"
-      done
+      applyall "${mfdir}"/kube-proxy"-ds.yaml"
 
       # Services
-      for manifest in {kube-dns,heapster,kubernetes-dashboard,metrics-server}; do
-          kubectl apply -f "${mfdir}/$manifest-svc.yaml"
-      done
+      applyall "${mfdir}"/{kube-dns,heapster,kubernetes-dashboard,metrics-server}"-svc.yaml"
 
       {{- if .Addons.Rescheduler.Enabled }}
-      kubectl apply -f "${mfdir}/kube-rescheduler-de.yaml"
+      applyall "${mfdir}/kube-rescheduler-de.yaml"
       {{- end }}
 
       # API Services
-      for manifest in {metrics-server,}; do
-          kubectl apply -f "${mfdir}/$manifest-apisvc.yaml"
-      done
+      applyall "${mfdir}/metrics-server-apisvc.yaml"
```

Comment:
@mumoshu there's some inconsistency here - `"${mfdir}"` and `"${mfdir}`. I'm getting this on latest master:
```
Dec 07 08:17:38 ip-10-0-30-104.eu-west-1.compute.internal retry[1983]: /opt/bin/install-kube-system: line 96: unexpected EOF while looking for matching `"'
Dec 07 08:17:38 ip-10-0-30-104.eu-west-1.compute.internal retry[1983]: Attempt 3 failed and there are no more attempts left!
Dec 07 08:17:38 ip-10-0-30-104.eu-west-1.compute.internal retry[1983]: /opt/bin/retry: line 12: return: can only `return' from a function or sourced script
```

Reply:
Probably your issue would be resolved by #1060. Would you mind trying?

Reply:
@mumoshu ok will try that one, thanks.

Reply:
@mumoshu yep that seems to have gotten the kube system deploys working again.